scripts/tests/versioner_hashes.yaml

```diff
@@ -171,3 +171,9 @@ a2e8814e28b97171ff4a06e5343063fed6763dd0: #26102
   libmesh: bad5b94
   moose-dev: 9831cd1
   wasp: 91987a3
+5327cf7e7946b97249ea8f1d65efa86ae24a8e33: #26300
+  mpich: 702900f
+  petsc: 684d2af
+  libmesh: bad5b94
+  moose-dev: 5b104ea
```

Comment:
why did moose-dev change?

Reply:
Because I modified `moose/conda/moose/*`

Reply:
Hmm, does it change every time with a new commit? I just automatically assumed to run the versioner tool. But I see now thats not necessary... I'll change it back.